app/javascript/src/Users/components/RoleInput.jsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+
+type Props = {
+  children?: React.Node
+}
+
+const RoleInput = ({ children }: Props) => (
```

Comment:
Tiny components like `AdminSection`, `FeatureAccessInput`, etc... IMHO, don't deserve to be in a separated file since they are used only by one Component... It's a bit annoying going to another file to see the definition of something that could've been defined in the same file, given the impact of these submodules. Anyway, I know your preferences

Reply:
The main reasons why I do this are tidiness: components are easier to locate by using the project explorer or even using a _go to file..._ feature using the components name; and readability: having less lines of code makes the component faster to read and easier to understand.

**But you're right, it's a bit overkill with such small components**. I will refactor it and find a better way to group them.

Reply:
IMO, if the piece of code is used ( or is expected to be used) in other files or is too large, it deserves to be a new component. If used in only one file, is not needed.